go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/googleapis/gnostic v0.0.0-20190624222214-25d8b0b66985 // indirect
 	github.com/gorilla/mux v1.7.3 // indirect
 	github.com/hashicorp/golang-lru v0.5.3
+	github.com/influxdata/influxdb v0.0.0-20180221223340-01288bdb0883
```

Comment:
i've also had this `go.mod` change on my machine, but let's keep it out of the repo to avoid unnecessary potential problems (as long as nothing breaks)